src/main/java/com/xiaomi/infra/pegasus/operator/rrdb_get_scanner_operator.java

```diff
@@ -17,7 +17,7 @@ public rrdb_get_scanner_operator(
       String tableName,
       get_scanner_request request,
       long partitionHash) {
-    super(gpid, tableName, partitionHash);
+    super(gpid, tableName, partitionHash, false);
```

Comment:
I suggest uses `allowBackupRequest` instead of `isWrite`. Because Scan is not appropriate for another secondary request. `rrdb_clear_scanner_operator` too.

Reply:
OK. What about multi-get and batch-get? Do they need backup request?

Reply:
- multiget 
- get (get includes batch-get)
- ttl
- exist